File: scratch/check_jobs.py
import os
from supabase import create_client

url = "https://bvlcfqoxlfmxbxpkuuhg.supabase.co"
key = "sb_secret_9QfRdUR3MoYPmyFOZ4r5KQ_eN2g7hfL"
supabase = create_client(url, key)

res = supabase.table("jobs").select("*").order("created_at", desc=True).limit(10).execute()
for job in res.data:
    print(f"Time: {job['created_at']} | Type: {job['type']} | Status: {job['status']} | Payload: {job['payload']}")

res_logs = supabase.table("system_logs").select("*").order("created_at", desc=True).limit(10).execute()
print("\n--- RECENT LOGS ---")
for log in res_logs.data:
    print(f"Time: {log['created_at']} | Level: {log['level']} | Message: {log['message']}")

File: scratch/check_jobs.mjs
import { createClient } from '@supabase/supabase-js';
import dotenv from 'dotenv';
dotenv.config({ path: '.env.local' });

const supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL || 'https://bvlcfqoxlfmxbxpkuuhg.supabase.co';
const supabaseKey = 'sb_secret_9QfRdUR3MoYPmyFOZ4r5KQ_eN2g7hfL';

const supabase = createClient(supabaseUrl, supabaseKey);

async function check() {
  const { data, error } = await supabase.from('jobs').select('*').eq('type', 'telegram_sync').order('created_at', { ascending: false }).limit(3);
  console.log('--- LATEST TELEGRAM JOBS ---');
  console.log(data);

  const { data: heartbeat } = await supabase.from('system_settings').select('*').eq('key', 'worker_heartbeat');
  console.log('--- WORKER HEARTBEAT ---');
  console.log(heartbeat);
}
check();
